### ISSUE TITLE
recommended way to adjust resolution in a Dataset class

### ISSUE THREAD

**Issue by jeremyjordan** [salu133445/muspy#14]
Hi, 

I'm really excited to find this library! Thanks for all your work on MusPy.

Is there a recommended way to adjust the resolution for all of the songs in a Dataset? I'd like to create a `FolderDataset` for a collection of MIDIs, but my MIDI files have a high resolution (480) which I'd like to adjust (e.g. 24) before training a model on the piano roll representation. I looked through the docs and don't see a recommended way to handle this.

By the way, when adjusting the resolution of a multitrack object, it's producing a lot of output due to [this line](https://github.com/salu133445/muspy/blob/e948db2785290a92c108b371cbcf1e285102a1db/muspy/base.py#L476). Is this intended?

**Comment by salu133445** [comment]
Hi, if you want to change the resolution for the whole dataset. You could do something as follows.

```python
dataset = muspy.FolderDataset("data/")
for i, music in enumerate(dataset):
    music.adjust_resolution(24)
    music.save(f"test/{i}.json")
```

Then, you can later load the dataset using `muspy.MusicDataset` (it's essentially a `muspy.FolderDataset` for MusPy JSON files).

```python
dataset = muspy.MusicDataset("test/")
for music in dataset:
    # do something
```

The print function in `muspy.Music.adjust_resolution` is unintended. Thanks for pointing it out. This has been fixed in ceff794.

**Comment by jeremyjordan** [comment]
Thanks! I also realized I could do the conversion with the `read` method override.

```
class MidFolderDataset(muspy.FolderDataset):
    _extension = "mid"

    def read(self, filename: Union[str, Path]) -> Music:
        """Read a file into a Music object."""
        return muspy.read_midi(self.root / filename).adjust_resolution(16)
```
